### DISCUSSION TITLE
How to install 3.12.1 version?

### DISCUSSION THREAD

**Discussion by amiria703** [coder/code-server#4617] [Q&A] [answered]
I have problem with node version which is solved in 3.12.1 but whenever I'm trying to install it using yarn or npm I get the error that this version does not exist:
![Screenshot_20211214-180045_Termux](https://user-images.githubusercontent.com/54179379/146017753-d4012942-d954-4b6e-a331-c4f101fa67ce.png)

**Comment by lg-nineoneninethreenine** [comment]
There is no 3.12.1, there is a pre-release of 4.0.0, which package are you downloading?

**Reply by amiria703** [reply]
> There is no 3.12.1, there is a pre-release of 4.0.0, which package are you downloading?

How to download that pre-release?
Or build from main branch?
I can't install 3.12.0 because nodejs-lts version is not 14 anymore and it's 16 so I can't install it anyway

**Reply by lg-nineoneninethreenine** [reply]
You can get Node 14 via the tarball, you could search for a tutorial in Google, works the same everywhere.

**Reply by jsjoeio** [reply]
> I can't install 3.12.0 because nodejs-lts version is not 14 anymore and it's 16 so I can't install it anyway

I suggest using a tool like `nvm` to install and use Node v14

**Comment by amiria703** [comment, marked as answer]
My problem solved by running this:
apt install binutils python && env FORCE_NODE_VERSION=16 yarn global add code-server --ignore-engines
